wowup-electron/src/app/pages/my-addons/my-addons.component.ts

```diff
@@ -102,6 +108,20 @@ export class MyAddonsComponent implements OnInit, OnDestroy {
           this._displayAddonsSrc.next(addons);
         });
       })
+
+    this._displayAddonsSrc
+      .pipe(takeUntil(this._destroyed$))
+      .subscribe((items: MyAddonsListItem[]) => {
+        this.dataSource.data = items;
+        this.dataSource.sortingDataAccessor = _.get;
+        this.dataSource.filterPredicate = (item: MyAddonsListItem, filter: string) => {
+          if (item.addon.name.trim().toLowerCase().indexOf(filter) >= 0) return true;
+          if (item.addon.latestVersion && item.addon.latestVersion.trim().toLowerCase().indexOf(filter) >= 0) return true;
```

Comment:
Lets make this item.addon.latestVersion && item.addon.latestVersion.trim().toLowerCase().indexOf(filter) >= 0 int a function, since its the same 3 times.

Reply:
Done. Created function in string.utils.ts.